package.json

```diff
@@ -2,26 +2,21 @@
   "version": "0.19.0",
   "name": "ambire-common",
   "description": "Common ground for the Ambire apps",
+  "scripts": {
+    "install:peers": "install-peers -f"
+  },
   "peerDependencies": {
     "@ambire/signature-validator": "^1.0.3",
     "@ensdomains/eth-ens-namehash": "^2.0.15",
+    "@types/react": "^17.0.0",
+    "@types/validator": "^13.7.0",
+    "@typescript-eslint/eslint-plugin": "5.5.0",
+    "@typescript-eslint/parser": "5.5.0",
     "@unstoppabledomains/resolution": "^7.1.4",
     "@walletconnect/client": "^1.7.1",
     "adex-protocol-eth": "git+https://git@github.com/AmbireTech/adex-protocol-eth.git",
     "bip44-constants": "^128.0.0",
-    "ethereumjs-util": "^7.1.3",
-    "ethers": "^5.5.2",
-    "react": "^17.0.0",
-    "url": "^0.11.0",
-    "validator": "^13.7.0"
-  },
-  "devDependencies": {
-    "@types/react": "17.0.39",
-    "@types/validator": "^13.7.0",
-    "@typescript-eslint/eslint-plugin": "5.5.0",
-    "@typescript-eslint/parser": "5.5.0",
-    "adex-protocol-eth": "git+https://git@github.com/AmbireTech/adex-protocol-eth.git#ef2f88f26824c64c1eb12c02a389baec2b2c3889",
-    "eslint": "8.3.0",
+    "eslint": "^7.11.0",
```

Comment:
Unfortunately, moving those as `peerDependencies` conflicts with the web app deps. Investigating.